docs/docs/how-to/functions/routing.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Routing
+---
+
+Function routing shares the same syntax as [page routing](/docs/reference/routing/file-system-route-api/).
+
+## Static Routing
+
+Both top-level and nested routes are supported.
+
+- `src/api/top-level.js` => `/api/top-level`
+- `src/api/directory/foo.js` => `/api/directory/foo`
+
+`index.js` files are routed at their directory path e.g. `src/api/users/index.js` => `/api/users`
+
+## Dynamic Routing
```

Comment:
Not yet supported in Cloud, right?

Reply:
not yet but close

Reply:
Do we need to note that then? I don't think the Fastly piece is being worked on at the moment (maybe next sprint).

Reply:
Yeah, if we're not going to be ready by docs launch then we should note it. Adding...